### PR TITLE
Azerbaijani language added

### PR DESCRIPTION
**Pull request by sevdimali**
## Description
I created Azerbaijani language. Tests not added

### Types of change
new language

## Checklist

- [x ] I have submitted the spaCy Contributor Agreement.
- [x ] I ran the tests, and all new and existing tests passed.
- [x ] My changes don't require a change to the documentation, or if they do, I've added all required information.

### CODE REVIEW

File: spacy/lang/az/examples.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+"""
+
+
+sentences = [
+    "Bu bir cümlədir.",
+    "Necəsən?",
+    "Qarabağ ordeni vətən müharibəsində qələbə münasibəti ilə təsis edilmişdir.",
+    "Məktəbimizə Bakıdan bir tarix müəllimi gəlmişdi.",
+    "Atılan növbəti mərmilər lap yaxınlıqda partladı.",
+    "Sinqapur koronavirus baxımından ən təhlükəsiz ölkələr sırasındadır.",
```
Comment (svlandeg) [inline spacy/lang/az/examples.py:14]: I see you've added some very relevant example sentences ;-)

**Review by svlandeg — COMMENTED** [review]
Thanks for the contribution, it's great to get minimal support for Azerbaijani into spaCy :-) 

It would be good to have some minimal tests, see for example https://github.com/explosion/spaCy/pull/7114/files. If you don't have the time to add these now, we'll take care of that for you after merging this PR.

File: spacy/lang/az/examples.py (hunk 1 above)
Comment (sevdimali) [reply spacy/lang/az/examples.py:14]: I just get text's from some news :) . If it makes sense I can replace them with simple sentences.

**Comment by sevdimali** [comment]
> Thanks for the contribution, it's great to get minimal support for Azerbaijani into spaCy :-)
> 
> It would be good to have some minimal tests, see for example https://github.com/explosion/spaCy/pull/7114/files. If you don't have the time to add these now, we'll take care of that for you after merging this PR.

Yes, I am planning to add tests shortly. Also, I will create tokenizer exceptions.

File: spacy/lang/az/examples.py (hunk 1 above)
Comment (svlandeg) [reply spacy/lang/az/examples.py:14]: No, it's fine! :-)

**Comment by svlandeg** [comment]
Awesome! Do you want to add that to this PR, or shall we go ahead and merge this, and you'll follow up with a new PR for the other functionality & tests?

**Comment by sevdimali** [comment]
> Awesome! Do you want to add that to this PR, or shall we go ahead and merge this, and you'll follow up with a new PR for the other functionality & tests?
Please merge this. I will create a new PR for tests and etc.

**Pull request merged**